docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/index.rst

```diff
@@ -12,7 +12,7 @@ Orbit is configured, you can unlink it from a |product| instance as well.
 
    Setting Up a Sandbox Instance <Setting_up_a_Sandbox_Instance>
    Setting Up a Full Orbit Installation <Setting_Up_a_Full_Orbit_Installation>
-   Unlink Orbit from |product| <Unlinking_Orbit>   
+   Unlinking Orbit <Unlinking_Orbit>
```

Comment:
not sure about this... @wabernatScality  do you have a workaround?
Having a variable in links does not work

Reply:
1) We shouldn't waste any time on Orbit links. It is on the chopping block for Zenko 2.0. 
2) This is an old link structure that we shouldn't be using. Ideally, the link should just be to the file name, and let Sphinx find the header.
3) Remember how I said "It's one or the other... I can't remember"? It's this one. If you link in index.rst, it has to be to a file name. Sphinx can't extrapolate the file name from a variable.

Reply:
Ok! I tried to change it in several ways but I can't even delete Unlinking_Orbit. When I do, nothing appears next to the bullet points of the index, so I'll leave it that way.